src/screens/login/index.js

```diff
@@ -118,6 +121,12 @@ class Login extends PureComponent {
             ? i18n.t('LOGIN.TO_REGISTER')
             : i18n.t('LOGIN.TO_LOGIN')}
         </Text>
+        <CheckBox
+          center
+          title="2 FA auth"
+          checked={isTwoFA}
+          onPress={() => this.setState({ isTwoFA: !isTwoFA })}
```

Comment:
move 
`() => this.setState({ isTwoFA: !isTwoFA })`
case after re-render, the function will be recreated every time.